automation/jenkins_build.sh

```diff
@@ -36,7 +36,7 @@ do
 		docker run --rm -e ARCH=$ARCH \
 						-e ACCESS_KEY=$ACCESS_KEY \
 						-e SECRET_KEY=$SECRET_KEY \
-						-e BUCKET_NAME=$BUCKET_NAME go-$ARCH-builder bash build.sh $GO_VERSION
+						-e BUCKET_NAME=$BUCKET_NAME go-$ARCH-builder bash -ex build.sh $GO_VERSION
```

Comment:
```
-e  Exit immediately if a command exits with a non-zero status.
-x  Print commands and their arguments as they are executed.
```

Rather than specifying them here, would it be better to put
```bash
set -e
set -x
```
directly into `build.sh` ?
(Hmmm, looks like `build.sh` already includes `set -e`)